windows/README.md

```diff
@@ -0,0 +1,57 @@
+# react-native-print Windows Implementation
+
+## Module Installation
+You can either use autolinking on react-native-windows 0.63 and later or manually link the module on earlier realeases.
+
+## Automatic install with autolinking on RNW >= 0.63
+react-native-print supports autolinking. Just call: `npm i react-native-print --save`
+
+## Manual installation on RNW >= 0.62
+1. `npm install react-native-print --save`
+2. Open your solution in Visual Studio 2019 (eg. `windows\yourapp.sln`)
+3. Right-click Solution icon in Solution Explorer > Add > Existing Project...
+4. Add `node_modules\react-native-print\windows\RNPrint\RNPrint.vcxproj`
+5. Right-click main application project > Add > Reference...
+6. Select `RNPrint` in Solution Projects
+7. In app `pch.h` add `#include "winrt/RNPrint.h"`
+8. In `App.cpp` add `PackageProviders().Append(winrt::RNPrint::ReactPackageProvider());` before `InitializeComponent();`
+
+## Windows print canvas
+
+On Windows, `react-native-print` needs an element in the visual tree to add the printable pages to.
+It will look for a XAML `Canvas` named `RNPrintCanvas` and use it.
+This needs to be added to the XAML tree of the screens where `react-native-print` is used.
+
```

Comment:
@jonthysell I wonder if we should bake logic into autolinking so that modules can check that certain conditions are met before they get autolinked, e.g. that there is this element. Or at least give a chance for modules to print out warnings / instructions of this kind.